docs/content/guides/getting-started/binding-to-data.md

```diff
@@ -54,43 +54,38 @@ const hot = new Handsontable(container, {
 ::: only-for react
 ::: example #example1 :react
 ```jsx
-import React, { Fragment, useEffect } from 'react';
+import React from 'react';
 import ReactDOM from 'react-dom';
 import { HotTable } from '@handsontable/react';
 import { registerAllModules } from 'handsontable/registry';
 
 // register Handsontable's modules
 registerAllModules();
 
-const ExampleComponent = () => {
-  const data = [
-    ['', 'Tesla', 'Nissan', 'Toyota', 'Honda', 'Mazda', 'Ford'],
-    ['2017', 10, 11, 12, 13, 15, 16],
-    ['2018', 10, 11, 12, 13, 15, 16],
-    ['2019', 10, 11, 12, 13, 15, 16],
-    ['2020', 10, 11, 12, 13, 15, 16],
-    ['2021', 10, 11, 12, 13, 15, 16]
-  ];
-  const hotSettings = {
-    data,
-    startRows: 5,
-    startCols: 5,
-    height: 'auto',
-    width: 'auto',
-    colHeaders: true,
-    minSpareRows: 1,
-    licenseKey: 'non-commercial-and-evaluation'
-  };
+const App = () => {
```

Comment:
@bulaj Could you change the `App` const back to `ExampleComponent`? [A while ago we decided to keep each example encapsulated inside an example component](https://handsoncode.slack.com/archives/C034WPQUG9W/p1652304579106529?thread_ts=1652274953.061309&cid=C034WPQUG9W), and other examples were transformed that way. It'd be good they're all consistent with each other.